server/createPage.js

```diff
@@ -0,0 +1,28 @@
+import {query} from './query.js';
+
+//Book creator endpoint
+
+export const createPage = async (req, res) => {
+  if (!req.body)
+    return res.status(406).send('What are you even trying to do here?');
+
+  const {pageDes} = req.body;
+
+  if (!pageDes) return res.status(406).send('Please enter a page description!');
+
+  let newNumber = 1;
+  // Figure out next page number
+  const [pageNumber] = await query(
+    `SELECT page_number FROM "pages_data" ORDER by page_number DESC LIMIT 1`
```

Comment:
You should be able to do something like `SELECT MAX(page_number) + 1 FROM "pages_data" WHERE bookId=$1`